pkg/skaffold/runner/v1/new.go

```diff
@@ -61,7 +62,9 @@ func NewForConfig(ctx context.Context, runCtx *runcontext.RunContext) (*Skaffold
 	isLocalImage := func(imageName string) (bool, error) {
 		return isImageLocal(runCtx, imageName)
 	}
-	labeller := label.NewLabeller(runCtx.AddSkaffoldLabels(), runCtx.CustomLabels(), runCtx.GetRunID())
+
+	// Always add skaffold-specific labels, except during `skaffold render`
+	labeller := label.NewLabeller(runCtx.Mode() != config.RunModes.Render, runCtx.CustomLabels(), runCtx.GetRunID())
```

Comment:
nit: can we keep the `AddSkaffoldLabels` method and move this logic there?

Reply:
sure